docs/docs/installation.md

```diff
@@ -15,3 +15,9 @@ brew install retr0h/tap/gilt
 ```bash
 go install github.com/retr0h/gilt@latest
 ```
+
+## Python
+
+```bash
+pip3 install python-gilt
+```
```

Comment:
@0xDEC0DE should I state anything else?

Reply:
I don't think so?  That's all one should need to do.